pkg/plugins/golang/v3/scaffolds/webhook.go

```diff
@@ -90,6 +92,22 @@ func (s *webhookScaffolder) Scaffold() error {
 		return err
 	}
 
+	if doHubScaffold {
+		if err := scaffold.Execute(
+			&api.Conversion{Force: s.force, Hub: true},
+		); err != nil {
+			return err
+		}
+	}
+
+	if doSpokeScaffold {
+		if err := scaffold.Execute(
+			&api.Conversion{Force: s.force, Spoke: true},
+		); err != nil {
+			return err
+		}
+	}
+
 	if doConversion {
 		fmt.Println(`Webhook server has been set up for you.
```

Comment:
Could we not here add a condition to improve the text message?
It is valid only the options are not used?

Reply:
fixed at [40988d1](https://github.com/kubernetes-sigs/kubebuilder/pull/3201/commits/40988d10f913dc18f0a5cdea715acccd36947b42)